### DISCUSSION TITLE
3004.388.7_1-gnuton1

### DISCUSSION THREAD

**Discussion by gnuton** [gnuton/asuswrt-merlin.ng#601] [Announcements]
##  GNUton's Asus Merlin changelog  ##

Date: June 10, 2024
Release: 3004.388.7_1-gnuton1

  - ADDED: XT kernel module for SDK 5.04 models 
         Allows Skynet and other addons to work
  - FIXED: RT-AX58U_V2: Fixed errors when changing 2.4 Ghz setting 
  - FIXED: TUF theme
  - FIXED: RT-AX82U V2 firmware upgrade not working

<hr /><em>This discussion was created from the release <a href='https://github.com/gnuton/asuswrt-merlin.ng/releases/tag/3004.388.7_1-gnuton1'>3004.388.7_1-gnuton1</a>.</em>

**Comment by danielnarbona** [comment]
Will try on my RT-AX92U. Thanks

**Reply by danielnarbona** [reply]
So far, so good. Will post if see something.

**Reply by yomammary** [reply]
So far so good over here too!

**Comment by Stefan1578** [comment]
GRANDE! Avevo già installato la beta senza problemi, grazie per il tuo lavoro!
DSL-AX82U

**Comment by vix80-max** [comment]
I would like to point out that the bug on the upstream line attenuation which remains at 0.00 decibels has not been corrected. How come ?

DSL-AX82U

**Reply by gnuton** [reply]
The details of the analysis about this issue have been reported in the ticket related to this issue

**Reply by Stefan1578** [reply]
Ho il tuo stesso modem router ma questo problema non ce l'ho 🤔

**Reply by vix80-max** [reply]
> Ho il tuo stesso modem router ma questo problema non ce l'ho 🤔

non so cosa dirti, questo bug lo avevano segnalato anche nella precedente release . Ma tu hai riavviato qualcosa dopo ? ma anche col riavvio a me rimane su 0,0 db

detto ciò, ma sei sicuro che sul tuo non lo fa? mi sembra strano a sto punto ...

**Reply by vix80-max** [reply]
> The details of the analysis about this issue have been reported in the ticket related to this issue

but it seems that on some Ax82u models like mine it is there... and on other models it isn't, I don't understand why this difference

**Comment by Podrock** [comment]
Only negative symptom that I've experienced on this build is that it seems to completely break apple's icloud private relay service on iPhone. I reverted back to 3004.388.6_2 and it started working again. I'm on the AX95Q

**Reply by KristianLH** [reply]
@Podrock - can you confirm that the former issue with mesh clients not connecting is solved in this release?

**Reply by Podrock** [reply]
Apologies I cannot confirm because I only have 1 node; Everything else I tested did work well - only negative symptom was that my icloud private relay keep disconnecting/reconnecting in a loop.

**Comment by fstlaneukraine** [comment]
Had to revert back to RT-AXE95Q_3004_388.6_0 as I seemed to have a lot of issues on 388.7 with 2.4Ghz. My Nest cameras and PetLibro Dockstream water fountain seemed to have the most issues and would drop multiple times a day.

**Reply by fstlaneukraine** [reply]
The frequency has gone done considerably (from notifications on my phone every 30 minutes to only once in the last 24hrs) but this morning I still got a message stating that one camera and the fountain disconnected again. Might have to go back one more version. I really doubt it's a lack of WiFi coverage as Google and PetLibro seem to suggest. I have two AXE95Q's , one upstairs and one downstairs - both are more or less in the center of the home - although the devices are on against the outside walls....hmmm...

**Reply by fstlaneukraine** [reply]
I went back to 388.7 but picked up two additional AXE95Q's for a total of 4. Evidently 40-45 devices (depending on what's powered on/being used) was too much for my existing two - although that was before I found out I could 'bind' devices to a specific node. So, I probably have overkill now but I have about 10 devices per node (including wired) now.
